### ISSUE TITLE
No Integration Documentation available?  

### ISSUE THREAD

**Issue by mehedi-iitdu** [sebacarrasco93/laravel-payku#1]
You just provided the library but did not mention the usage of it anywhere.

**Comment by sebacarrasco93** [comment]
Originally it was made for personal projects (because Payku is only a Chilean "payments gateway"). But if you want to use it, I would be happy to create the corresponding documentation 🙂

**Comment by mehedi-iitdu** [comment]
Yes, great. I'm already using it. I want to know how to enable multiple gateways. Right now it's redirecting to WebPayPlus directly.

**Comment by sebacarrasco93** [comment]
Hello @mehedi-iitdu , I arrived about 3 years later, but I had been very busy. I hadn't realized how many users this package had, so I'm improving it, if you notice, it will now have all the methods available from the official documentation from Payku.
